docs/src/quickstart.md

```diff
@@ -160,6 +161,63 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+With a few more steps, we can use Turing to compute additional useful groups to add to the [`InferenceData`].(@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+To draw from the prior and posterior predictive distributions we can instantiate a "predictive model", i.e. a Turing model but with the observations set to `missing`, and then calling `predict` on the predictive model and the previously drawn samples:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(J, similar(y, Missing), σ)
+# and then sample!
+prior_predictive = predict(param_mod_predict, prior)
+posterior_predictive = predict(param_mod_predict, turing_chns)
+```
+And to extract the elementwise log-likelihoods, which is useful if you want to compute metrics such as [`loo`](@ref),
+```@example quickstart
+loglikelihoods = Turing.elementwise_loglikelihoods(param_mod, turing_chns)
+```
+This can then be included in the `from_mcmcchains` call from above:
+```@example quickstart
+# Convert into format compatible with ArviZ.jl
+using LinearAlgebra
+# Ensure the ordering of the loglikelihoods matches the ordering of `posterior_predictive`
+loglikelihoods_vals = getindex.(Ref(loglikelihoods), string.(keys(posterior_predictive)))
```

Comment:
Would this be more digestible (I don't know about the variable name `ynames`, naming things is hard).
```suggestion
ynames = string.(keys(posterior_predictive))
loglikelihoods_vals = getindex.(Ref(loglikelihoods), ynames)
```